blockonomics-woocommerce.php

```diff
@@ -401,89 +408,6 @@ function bnomics_async_scripts($url)
 register_activation_hook( __FILE__, 'blockonomics_activation_hook' );
 add_action('admin_notices', 'blockonomics_plugin_activation');
 
-global $blockonomics_db_version;
-$blockonomics_db_version = '1.0';
-
-function blockonomics_create_table() {
```

Comment:
These changes have been merged into master and should not be removed in the PR. Please merge master into this branch.

Reply:
Now master should be merged into this branch.